api_specifications/filestore-1.1.yaml

```
@@ -0,0 +1,387 @@
+      -  Uploads different kinds of files to server including images and different document types.
+  contact:
+    name: eGovernments Foundation
+    email: contacts@egovernments.org
+  version: 2.0.0
+servers:
+- url: /filestore/v1
+paths:
+  /files:
+    post:
+      tags:
+      - Upload File
```

yamllint

[error] wrong indentation: expected 8 but found 6

(indentation)